install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,209 @@
+networking capabilities.
+
+NSX-T supports Virtual Machine, Bare-metal, and container workloads across multiple clusters. This allows
+organisations to have complete visibility using a single SDN across the entire environment.
+
+
+See xref:../architecture/networking/network_plugins.adoc#nsx-sdn[Networking]
+for more information on how NSX-T is integrated with {product-title}.
+
+[[nsx-t-sdn-operations-workflow]]
+== Example Topology
+
+There is a *T0 router* on the top that connects physical with virtual world. We also have *T1 router* acting as a default gateway for the OpenShift VMs.
+
+Those VMs have *two vNICs* each. One vNIC is connected to Management Logical Switch for accessing the VMs. The second vNIC is connected to a dump Logical Switch and is used by nsx-node-agent to uplink the POD networking. For further details, we strongly suggest reading the link:https://docs.vmware.com/en/VMware-NSX-T-Data-Center/2.3/nsxt_23_ncp_OpenShift.pdf[NSX Container Plug-in for OpenShift].
+
+The LoadBalancer used for configuring OpenShift Routes plus all project’s *T1 routers* and *Logical Switches* are created automatically later when we install OpenShift. In this topology we use the default OpenShift HAProxy Router for all infrastructure components such as Grafana, Prometheus, Console, Service Catalog and others.
+
+This means that the DNS records for the infra components need to point the infrastructure node IP addresses since the HAProxy uses the host network namespace.
+This works well for infra routes but in order to avoid exposing the infra nodes management IPs to the outside world we will be deploying application specific routes to the NSX-T LoadBalancer.
+
+The topology here assumes 3 x OpenShift master virtual machines and 4 x OpenShift worker virtual machines(two for infrastructure and two for compute).
+
+[[nsx-t-sdn-installation]]
+== Installation
+
+Prerequisites:
+
+* ESXi hosts requirements:
+** ESXi servers that host {product-title} node VMs must be NSX-T Transport Nodes.
+
+Example of High-Available environment: +
+image:images/nsxt-transportnodes.png[NSX Transport Nodes]
+
+* DNS requirements:
+** *You must add* to your DNS server a new entry with a wildcard to the 'infra' nodes, this allows load balancing by NSX-T or other third-party LB. In the 'hosts' file below, it's defined by 'openshift_master_default_subdomain' variable.
+** You have also to update your DNS server accordingly with: 'openshift_master_cluster_hostname' & 'openshift_master_cluster_public_hostname' variables.
+
+* Virtual Machine requirements:
+** {product-title} node VMs must have *two vNICs*:
+** Management vNIC connected to the *Logical Switch* that is uplinked to the management *T1 router*.
+** The second vNIC on all VMs needs to have two Tags in *NSX-T* in order 'nsx-container-plugin' (NCP) to know which port needs to be used as a parent VIF for all PODs running in the particular OpenShift node. Tags need to be as following:
```

Comment:
@RobbieJVMW In _Tags in *NSX-T* in order_ what does _Tags in NSX-T_ mean?

Reply:
Tag is essentially a label. NSX Tag's are how the nsx platform manages relationships between components.

Reply:
I saw that better when I got the image to appear in my preview. Thanks!